pkg/conversion/code_generation_test.go

```diff
@@ -145,7 +146,13 @@ func TestNoManualChangesToGenerateConversions(t *testing.T) {
 		existingFunctions, existingNames := bufferExistingConversions(t, fileName)
 		generatedFunctions, generatedNames := generateConversions(t, version)
 
-		compareBuffers(t, existingFunctions, generatedFunctions)
-		compareBuffers(t, existingNames, generatedNames)
+		functionsTxt := fmt.Sprintf("%s.functions.txt", version)
+		ioutil.WriteFile(functionsTxt, generatedFunctions.Bytes(), os.FileMode(0644))
+
+		namesTxt := fmt.Sprintf("%s.names.txt", version)
+		ioutil.WriteFile(namesTxt, generatedNames.Bytes(), os.FileMode(0644))
+
+		compareBuffers(t, functionsTxt, existingFunctions, generatedFunctions)
+		compareBuffers(t, namesTxt, existingNames, generatedNames)
```

Comment:
But perhaps clean up in the success case still since they're only meant to be used in the failure case